ludwig/api.py

```diff
@@ -538,7 +539,7 @@ def on_epoch_end(self, trainer, progress_tracker, save_path):
                 # save description
                 if self.backend.is_coordinator():
                     description = get_experiment_description(
-                        self.config_obj.to_dict(),
+                        scrub_creds(self.config_obj.to_dict()),
```

Comment:
To be even stricter, can we make `.to_dict()` call `scrub_creds()`?

I imagine that any callers of `confict_obj.to_dict()` can do without rendered credentials. Consumers of credentials via config_obj should use the backend object directly, or via something like `config_obj.backend.credentials`.

Reply:
I can def make the change, but I don't know if that'll break something. Let's find out.